CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 3.1.0
+* Added `direct_only` option to `PolicyMachineStorageAdapter::ActiveRecord::accessible_objects` to fetch only directly assigned objects.
+
```

Comment:
FWIW, when I bumped the minor version I wasn't sure what to bump.. looks like for the pluck method they just bumped tiny so perhaps this should be 3.0.3 rather than 3.1.0?  I don't really have an opinion but wanted to point out for discussion.

Reply:
Think that's fine for our purposes